### ISSUE TITLE
Problem about the bad results

### ISSUE THREAD

**Issue by eedavidwu** [JiaRenChang/PSMNet#152]
Hi JiaRen:
   I test the pretrained model but get the bad results as other people. (My torchvision=0.2.0)
   Then I train the model in scene flow. (10 epoch with loss at about 5)
   Then the results are still bad...
   I will try more epoch, but It this results normal?
   Seems a bit strange...
![disp_0001](https://user-images.githubusercontent.com/43716059/64612189-c960cc80-d405-11e9-9bbf-900ada493818.png)
